### PR TITLE
Add count_bits_xxx functions

### PR DESCRIPTION
**Pull request by rge-exafore**

### CODE REVIEW

File: include/libswiftnav/bits.h [hunk 1]
```diff
@@ -23,5 +23,6 @@ void setbits(u8 *buff, u32 pos, u32 len, s32 data);
 void bitcopy(void *dst, u32 dst_index,
              const void *src, u32 src_index, u32 count);
 void bitshl(void *buf, u32 size, u32 shift);
+u8 get_set_bit_number(u32 v);
```
Comment (adel-mamin) [inline include/libswiftnav/bits.h]: get_set_bit_number -> get_bit_count / getbitcount ?
get_set_ prefix looks a bit confusing to my taste.

**Comment by adel-mamin** [comment]
Looks good to me.

**Comment by rge-exafore** [comment]
@gsmcmullin please review

File: src/bits.c [hunk 2]
```diff
@@ -198,4 +201,20 @@ void bitcopy(void *dst, u32 dst_index, const void *src, u32 src_index,
...
 
+/**
+ * Count number of bits set in 32 bit value
+ *
+ * \param[in]     v   32 bits value
+ * \param[in]     bv  1 or 0 - which value to count
+ *
+ * \return        Number of bits set to one or zero.
+ */
+u8 get_bits_number(u32 v, u8 bv)
```
Comment (jacobmcnamee) [inline src/bits.c]: Can we rename this to `count_bits_u32()` or something a little more descriptive?

Comment (jacobmcnamee) [inline src/bits.c]: And if it might ever be useful... Perhaps `count_bits_u16()` and `count_bits_u8()` which hopefully reuse the same code :)

Comment (rge-exafore) [inline src/bits.c]: Done.

**Comment by rge-exafore** [comment]
@valeri-atamaniouk please review

**Comment by valeri-atamaniouk** [comment]
Looks good

**Comment by ljbade** [comment]
Looks good to me (in case the others miss this)

**Comment by ljbade** [comment]
Be sure to also put in private

**Pull request merged**